triton/tut/connecting.rst

```diff
@@ -271,34 +278,59 @@ Exercises
    ``top`` works almost as well).  What else can you learn about the
    node?
 
+   .. solution::
+
+      The node was last rebooted 18 days ago, and the load average
```

Comment:
Maybe clarify the solution a bit by appending to the start something like: "You should see something like this. From this example output we can tell that..."